samples/image-metadata/src/components/FocalPoint/FocalPoint.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { styles } from './styles';
+
+export const FocalPoint = ({ focalPoint }) => (
+  <div
+    style={{
+      width: '32px',
```

Comment:
This could be extracted since you use it 4 times here

Reply:
Done